### PR TITLE
[Web] Update getting-started documents for Tizen 5.0

### PR DESCRIPTION
**Pull request by journey2w**
### Change Description ###

Update screenshots of Web Getting Started documents for Tizen 5.0.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/450

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/450

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/450

File: docs/application/web/getting-started/wearable-widget/first-app-widget.md [hunk 1]
```diff
@@ -81,7 +80,7 @@ To create the application project:
 
     3.  Select the **Web Application** application type and click **Next**.
 
-        ![Selecting the application type](media/create_project_wizard_app_web_wearable.png)
+        ![Selecting the application type](media/create_project_wizard_app_ww.png)
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/web/getting-started/wearable-widget/first-app-widget.md]: Just for the check - if we are not going to change the images for widget, too, (not watch), how about reverting this create_project_wizard_app_ww.png back to "4.0" version?

As I commented,
 - other images in this page are on "4.0" version.
 - this image is containing "5.0" information alone.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/450

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

**Pull request merged**